test/unit/index.js

```diff
@@ -14,20 +14,48 @@ test('that translation files are valid json', t => {
       jsonFiles.map(name),
       'each xml file has a corresponding json file'
     )
+    const testFile = file => {
+      let json = null
+      try {
+        json = require(file)
+      } catch (e) {
+        console.error(e.message)
+        return false
+      }
+      return Object.keys(json).every(k1 => {
+        const v1 = json[k1]
+        return Object.keys(v1).every(k2 => {
+          const v2 = v1[k2]
+          if (typeof v2 !== 'string') {
+            console.error(
+              `> ${JSON.stringify(v2)} not a string (${file} -> ${k1} -> ${k2})`
+            )
+            return false
+          }
+
+          function testString (str) {
+            const regex = new RegExp(str, 'g')
+            const match = regex.exec(v2)
+            if (match) {
+              console.error(`> ${JSON.stringify(v2)} contains ${str} (${file} -> ${k1} -> ${k2}) (index: ${match.index})`)
+              return false
+            }
+            return true
+          }
+
+          if (!testString('\\\\n')) return false
+          if (!testString("\\\\'")) return false
+          if (!testString('\\\\\\"')) return false
+          if (!testString('\\\\')) return false
```

Comment:
And it shouldn't contain these.

Reply:
Could probably refactor this into one `return` statement. Up to you guys.

Reply:
It's fine with multiple returns to me